scripts/addRuleToExportIndex.ts

```diff
@@ -45,14 +53,15 @@ function transformer(file, api, options) {
         // Manually sort the export statements alphabetically
         const sortedExports = updatedExportStatements
             .nodes()
-            .map(node => {
+
+            .map((node: any) => {
```

Comment:
Let's avoid 'any' and use the types. `import { TSESTree } from "@typescript-eslint/utils";` that package should have the right types.